jans-cedarling/bindings/cedarling_python/src/config/bootstrap_config.rs

```diff
@@ -5,520 +5,160 @@
-            idt_sub_validation,
-            idt_exp_validation,
-            idt_iat_validation,
-            idt_aud_validation,
-            userinfo_iss_validation,
-            userinfo_sub_validation,
-            userinfo_aud_validation,
-            userinfo_exp_validation,
-            id_token_trust_mode,
-            lock,
-            lock_master_configuration_uri,
-            dynamic_configuration,
-            lock_ssa_jwt,
-            audit_log_interval,
-            audit_health_interval,
-            audit_health_telemetry_interval,
-            listen_sse,
+            inner,
+            source: Some(source),
         })
     }
 
-    pub fn disable_all_jwt_validation(&mut self) {
-        self.jwt_sig_validation = "disabled".to_string();
-        self.jwt_status_validation = "disabled".to_string();
-
-        self.at_iss_validation = "disabled".to_string();
-        self.at_jti_validation = "disabled".to_string();
-        self.at_nbf_validation = "disabled".to_string();
-        self.at_exp_validation = "disabled".to_string();
+    #[staticmethod]
+    pub fn load_from_file(path: &str) -> PyResult<Self> {
+        let inner = cedarling::BootstrapConfig::load_from_file(path).map_err(|e| match e {
+            cedarling::BootstrapConfigLoadingError::ReadFile(file_name, e) => {
+                PyOSError::new_err(format!("Error reading file `{}`: {}", file_name, e))
+            },
+            err => PyValueError::new_err(err.to_string()),
+        })?;
+        Ok(Self {
+            inner,
+            source: None,
+        })
+    }
 
-        self.idt_iss_validation = "disabled".to_string();
-        self.idt_sub_validation = "disabled".to_string();
-        self.idt_exp_validation = "disabled".to_string();
-        self.idt_iat_validation = "disabled".to_string();
-        self.idt_aud_validation = "disabled".to_string();
-        self.id_token_trust_mode = "none".to_string();
+    #[staticmethod]
+    pub fn load_from_json(config_json: &str) -> PyResult<Self> {
+        let inner = cedarling::BootstrapConfig::load_from_json(config_json)
+            .map_err(|e| PyValueError::new_err(e.to_string()))?;
+        Ok(Self {
+            inner,
+            source: None,
+        })
+    }
 
-        self.userinfo_iss_validation = "disabled".to_string();
-        self.userinfo_aud_validation = "disabled".to_string();
-        self.userinfo_sub_validation = "disabled".to_string();
-        self.userinfo_exp_validation = "disabled".to_string();
+    #[getter]
+    pub fn get_policy_store_local_fn(&mut self) -> Option<String> {
+        self.source
+            .as_ref()
+            .map(|s| s.policy_store_local_fn.clone())?
     }
-}
 
-fn get_required<'py, T>(options: &Bound<'py, PyDict>, key: &str) -> PyResult<T>
-where
-    T: FromPyObject<'py>,
-{
-    options
-        .get_item(key)?
-        .ok_or_else(|| PyKeyError::new_err(format!("Missing required config: `{}`", key)))?
-        .extract()
-}
+    #[setter]
```

Comment:
I think we can use `clasmethod`  in this case we don't need `self` and can store `source` without `Option`

Reply:
the reason why `source` is an `Option` is because we don't have access to the `BootstrapConfigRaw` when we call `BootstrapConfig::load_from_json`.


I guess i can remove the `source` and just have the `BootstrapConfig` but I didn't do that because there are tests that try to edit the bootstrap config and I didn't really want to touch them. For example: 
```py
def test_load_policy_store_ok(sample_bootstrap_config):
    # map fixture to variable with shorter name for readability
    config = sample_bootstrap_config

    policy_store_location = join(TEST_FILES_PATH, "policy-store_ok.yaml")
    config.policy_store_local_fn = policy_store_location # CONFIG IS GETTING EDITED HERE

    # initialize cedarling
    Cedarling(config)
```

Reply:
Let me try another solution without having two structs. Though I'll probably have to edit the tests then remove the setters.